easycv/predictors/detector.py

```diff
@@ -36,6 +35,184 @@
     from easycv.thirdparty.mtcnn import FaceDetector
 
 
+@PREDICTORS.register_module()
+class DetectionPredictor(PredictorInterface):
```

Comment:
by calling it detection predictor， does it support all the detection models?

Reply:
In addition to yolox, xinyi is still being reconstructed and can be used in a unified manner after the reconstruction is complete